src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+
+type Props = {
+  todos: Todo[];
+  onDelete: (todoToDelete: Todo) => void;
+  tempTodo: Todo | null;
+  updateTodo: (todoId: number, completed: boolean) => Promise<void>;
+  handleTitleEdit: (todoId: number, titles: string) => Promise<void>;
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  onDelete,
+  tempTodo,
+  updateTodo,
+  handleTitleEdit,
```

Comment:
You pass these props down without using them in this component. Better to use Context in this case. It is a suggestion for improvement, not a mistake. So if you're aware to do it, go forward. If not - it's ok